pkg/transformers/config/defaultconfig/commonlabels.go

```diff
@@ -144,19 +144,4 @@ commonLabels:
   create: false
   group: policy
   kind: PodDisruptionBudget
-
-- path: spec/podSelector/matchLabels
```

Comment:
This should still be allowed, no? It's referencing resources in the namespace.

Reply:
It's still allowed just not the default behaviour.